subsys/net/l2/ethernet/gptp/Kconfig

```diff
@@ -226,4 +226,41 @@ config NET_GPTP_STATISTICS
 	  Enable this if you need to collect gPTP statistics. The statistics
 	  can be seen in net-shell if needed.
 
+config NET_GPTP_SERVO_STEP_THRESHOLD
+	int "Set slave clock jump threshold (ns)"
```

Comment:
Should we mention servo here like in the other added config options?

Reply:
If we use a servo controller, we must expose some parameters to the user for configuration to meet the actual situation. Do you have a better suggestion for this?

Reply:
@chencaidy @jukkar I propose that we keep everything that relates to the implementation of a specific servo implementation in a separate namespace (common prefix for Kconfig entries, functions, structs, global/static variables, etc) and possibly even separate files (.c and .h files). If separate files seem to be overengineering in this case, separate namespacing and clear separation of concerns inside the media independent layer would also be fine IMO.